samples/bluetooth/mesh/sensor_client/README.rst

```diff
@@ -0,0 +1,182 @@
+.. _bluetooth_mesh_sensor_client:
+
+Bluetooth: Mesh sensor client
+#############################
+
+The Bluetooth Mesh sensor client sample demonstrates how to set up a basic Bluetooth Mesh :ref:`sensor client <bt_mesh_sensor_cli_readme>` model application that gets sensor data from one :ref:`sensor server <bt_mesh_sensor_srv_readme>` model.
+
+.. note::
+    This sample must be paired with :ref:`bluetooth_mesh_sensor_server` to show any functionality. A client has no sensor data and is dependent on a server to provide it.
+
+Overview
+********
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling Mesh models, :file:`model_handler.c`.
+
+The following Mesh sensor types are used in this sample:
```

Comment:
Could you provide a line of explanation after this list or in line 7 (which would be better, I think) about why these various sensor types are used in this sample?

Reply:
For example "Four different sensor types are used to showcase different ways for the server to publish data. In addition, usage of both single-channel sensors types and series sensor types are shown (reference: http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/bluetooth/mesh/sensor.html?#bt-mesh-sensor-types-channels)."

Reply:
Thanks.